### PR TITLE
Move EKS deployer to within kubetest

### PR DESCRIPTION
**Pull request by shyamjvs**
Maintenance of eks jobs will get very complicated as we rely more on env vars and presets. We should prefer using flags instead. This is a step towards that change.

/cc @gyuho 
fyi - @krzyzacy @BenTheElder

### CODE REVIEW

**Comment by gyuho** [comment]
/lgtm

Thanks @shyamjvs !

**Comment by gyuho** [comment]
^ @shyamjvs

**Comment by shyamjvs** [comment]
@krzyzacy That sgtm - kind of aligns with what I was thinking. Can we begin by maybe migrating some of the envs to flags?

File: kubetest/eks.go [hunk 1]
```diff
@@ -14,17 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package eks implements 'kubetest' deployer interface.
+// Package main / eks.go implements kubetest deployer interface for EKS.
```
[the remark this answers is not in the document]
Comment (BenTheElder) [reply kubetest/eks.go]: see below: don't use flags - use a config file or a structured flag

adding more flags to main is the same problem, the flag-variable struct definition is like 100 lines last I checked! this is not reasonable for anyone working on kubetest.

[the remark this answers is not in the document]
Comment (BenTheElder) [reply kubetest/eks.go]: I don't believe the maintenance cost of the configs is comparable to the mess within kubetest currently, and changes to kubetest potentially affect _all_ of the jobs.

**Pull request merged**